packages/safe-apps-wagmi/package.json

```diff
@@ -22,15 +22,15 @@
   "author": "Safe (https://gnosis-safe.io)",
   "license": "MIT",
   "dependencies": {
-    "@gnosis.pm/safe-apps-provider": "^0.15.0",
-    "@gnosis.pm/safe-apps-sdk": "^7.7.0"
+    "@gnosis.pm/safe-apps-provider": "^0.15.1",
+    "@gnosis.pm/safe-apps-sdk": "^7.8.0"
   },
   "peerDependencies": {
-    "@wagmi/core": ">= 0.3.8 < 0.8.0"
+    "wagmi": "^0.10.10"
```

Comment:
why does it change from `@wagmi/core` to `wagmi`? iirc `wagmi` is a react specific package but the connector doesn't care if it will be used in react or other framework, that's why it used `@wagmi/core`

Reply:
Understood, reverting to wagmi/core but bumping the version

Reply:
@mikhailxyz I did push the change let me know if somethig else is required

Reply:
I've just added a minor comment about version range for wagmi dep. Going to test the PR now

Reply:
I cannot get it working in the wagmi example app, I'm getting an error:
```
Type 'SafeConnector' is not assignable to type 'Connector<any, any, any>'.
  Property 'onAccountsChanged' is protected but type 'SafeConnector' is not a class derived from 'Connector<Provider, Options, Signer>'.
```

I also had to remove the `defaultChains` import: 
https://wagmi.sh/react/migration-guide#chain-exports